src/crimson/os/seastore/random_block_manager/nvme_block_device.cc

```diff
@@ -18,6 +18,105 @@ namespace {
     return crimson::get_logger(ceph_subsys_seastore_tm);
   }
 }
+namespace crimson::os::seastore::random_block_device {
+#include "crimson/os/seastore/logging.h"
+SET_SUBSYS(seastore_device);
+
+RBMDevice::mkfs_ret RBMDevice::mkfs(device_config_t config) {
+  LOG_PREFIX(RBMDevice::mkfs);
+  super.start = 0;
+  super.end = get_available_size();
+  super.block_size = get_block_size();
+  super.size = get_available_size();
```

Comment:
Added TODO comment---I will add this part along with a real device (not ephemeral) test because it seems that open_device() and check_create_device() are needed. Do you think this part is required for now?